filters/tracing/tag_test.go

```diff
@@ -95,19 +226,69 @@ func TestTracingTag(t *testing.T) {
 				},
 			},
 		},
+		"test_tag",
 		"true",
+	}, {
+		"tag from response if condition is true",
+		NewTagFromResponseIfStatus(),
+		[]interface{}{"Error", "true", 500, 599},
```

Comment:
I now prefer to use eskip in tests like e.g. here https://github.com/zalando/skipper/pull/3022 - this way it will catch any type conversion issues https://github.com/zalando/skipper/pull/3202/files#r1725191431